.github/CONTRIBUTING.md

```diff
@@ -1,8 +1,11 @@
 **The Azure IoT SDKs team wants to hear from you!**
 
 - [Need Support?](#need-support)
-- [File a bug](#file-a-bug)
+- [File a bug (code or documentation)](#file-a-bug-code-or-documentation)
 - [Contribute code and/or documentation](#contribute-code-andor-documentation)
+    - [Commit messages](#commit-messages)
```

Comment:
>     [](start = 0, length = 4)

Nit:Spacing here and in the next few lines is odd.

Reply:
The spacing here is autoformatted from a markdown editor. It makes sense since it is a sub-bullet.

---
In reply to: [372369664](https://github.com/Azure/azure-iot-sdk-node/pull/682#discussion_r372369664) [](ancestors = 372369664)